src/main/java/eu/xenit/gradle/alfrescosdk/tasks/AmpSourceSetConfiguration.java

```diff
@@ -28,17 +30,19 @@ default AmpSourceSetConfiguration fileMapping(Map<String, String> fileMappingPro
 
     AmpSourceSetConfiguration fileMapping(Action<? super Properties> configure);
 
-    AmpSourceDirectories getConfig();
+    SourceDirectorySet getConfig();
 
     default AmpSourceSetConfiguration config(Action <? super AmpSourceDirectories> configure) {
-        configure.execute(getConfig());
+        getConfig().getSourceDirectories().forEach(dir -> {
```

Comment:
I don't get what this is doing.

Reply:
If I understand correctly, this method is no longer needed?

Reply:
I think it might, since it provides a default configuration